src/js/nizovi.js

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable no-unused-vars */
+// 1. zadatak
+const slova = ['s', 'a', 'e']
+// 2. zadatak
+const brojevi = [1, 2, 3]
+
```

Comment:
Ovde si namerno preskočio 3. zato što je veliki, dugačak i dosadan? :smile:

Reply:
Planirao sam i njega eventualno da uradim, ali kad sam isao redom, da preskocio sam ga :D

Reply:
Branko, ovde se misli na niz imena(10 elem), pa niz prezimena, pa niz automobila itd. ?